crawler/src/main/java/com/brandwatch/ahmedanalytics/crawler/CrawlerJob.java

```diff
@@ -14,24 +17,34 @@
 
 @Component
 public class CrawlerJob {
+    private static final ObjectMapper mapper = new ObjectMapper();
```

Comment:
Perhaps you could consider using different de/serialisers, so that you don't need this ObjectMapper. @YNedderhoff mentioned (this in one of my PRs)[https://github.com/amcmurray-bw/minianalytics/pull/11/files/06676fe2436405a5ab612d33497a14bd62d96803#r214692904].

Reply:
A new instance of ObjectMapper is created here. It should be injected instead.

Alternatively it could be injected into json de/serializer classes and they can be used to configure Producer/Consumer. I am fine with both options. You can explore them and tell us which one you think is better and why.

Reply:
Done 👍